pkg/client/proxy_replica.go

```diff
@@ -204,12 +204,15 @@ func (c *ProxyClient) ReplicaRemove(dataEngine, serviceAddress, engineName, repl
 	input := map[string]string{
 		"serviceAddress": serviceAddress,
 		"engineName":     engineName,
-		"replicaAddress": replicaAddress,
 	}
 	if err := validateProxyMethodParameters(input); err != nil {
 		return errors.Wrap(err, "failed to remove replica for volume")
 	}
 
+	if replicaAddress == "" && replicaName == "" {
+		return fmt.Errorf("failed to remove replica for volume: replica address and name are both empty")
```

Comment:
Maybe can be useful to log the volume name too.

Reply:
longhorn-manager will receive the error and log the error with the volume name, so we don't need to add the volume name here.